playbooks/roles/monit/templates/monitrc.j2

```diff
@@ -5,9 +5,11 @@ set idfile /var/lib/monit/id
 set statefile /var/lib/monit/state
 
 set httpd port 2812 and
-    use address localhost  # only accept connection from localhost
-    allow localhost        # allow localhost to connect to the server
+    use address localhost  # Bind monit to localhost
 
+    # Allow any ip to connect to monit. Binding monit to localhost
+    # and requiring basic auth (through nginx) should be sufficient to protect this interface.
+    allow 0.0.0.0/0.0.0.0 # allow any ip to connect to monit.
```

Comment:
The whitelist in this context is referring to the "Client IP" that you're accessing monit with. 

So if you're striesand IP was 25.10.136.211, with the current rules it would be blocked. If we added `{{ streisand_ipv4_address }}` you should be able to access monit if `eth0`on the server and `{{ streisand_ipv4_address }}` are the same 

Regarding the ` I realized that this may not always work`  I tested adding `{{ streisand_ipv4_address }}` with digitial ocean and it worked fine. When I tested with Azure it didn't work. When I did more digging, this was because on digital ocean `eth0 == {{ streisand_ipv4_address }}`. On Azure (probably GCE, and AWS) `eth0 != {{ streisand_ipv4_address }}`. `eth0` is a private IP. 

On Azure the "Client IP" that was trying to access monit, was equal the private IP. I immediately thought we could try set_real_ip_from, and or X-Forwared-For.. but decided maybe i'm making this more difficult than it needed to be. Hence the 

>It then occurred to me that if we consider nginx + basic auth good enough to protect streisant vpn connection info; It's good enough protect the monit url path